etc/openqa/workers.ini

```diff
@@ -23,3 +23,8 @@
 
 #[openqa.example.host]
 # SHARE_DIRECTORY = /var/lib/openqa/example
+# In case caching of assets and tests needs to be enabled
+# CACHEDIRECTORY and TESTPOOLSERVER need to be enabled
+# CACHEDIRECTORY = /var/lib/openqa/cache
+# TESTPOOLSERVER = rsync://deimos.suse.de/openqa-tests
```

Comment:
ok, let me retry: I am sorry, I don't understand what `TESTPOOLSERVER` means just by reading the name and I am saying this because I think others will have the same problem. I would appreciate if `TESTPOOLSERVER` is described in more details.

@coolo What would be your approach how to phrase a request like the above into an open question which makes the requestee think but also incorporate the suggestions?